packages/mwp-app-render/src/util/getClient.js

```diff
@@ -1,7 +1,7 @@
 import ApolloClient, { InMemoryCache } from 'apollo-boost';
 import fetch from 'isomorphic-fetch';
 
-const GRAPHQL_ENDPOINT = process.env.GRAPHQL_ENDPOINT || 'https://api.meetup.com/gql';
+const GRAPHQL_ENDPOINT = process.env.GRAPHQL_ENDPOINT || 'https://www.meetup.com/gql';
```

Comment:
note: I've tried just setting `GRAPHQL_ENDPOINT`  in mup-web but it doesn't work for client side apollo requests